gradle/verification-metadata.xml

```diff
@@ -1270,6 +1270,11 @@
             <sha256 value="9b733854e81a1187c5262468cd962d9afdd9bfc7d77c8d160e45249dcd4c64c9" origin="Generated by Gradle" reason="Artifact is not signed"/>
          </artifact>
       </component>
+      <component group="org.gradle.kotlin" name="gradle-kotlin-dsl-plugins" version="4.0.6">
```

Comment:
Shouldn't this replace the old entry for `kotlin-dsl-plugins`?

Reply:
This PR doesn't update the wrapper on `master`. The `kotlin-dsl` plugin on `master` has diverged with the property assignment overload support. I could have removed that entry from `master` because it will be of no use, good catch.

Following this PR I need to publish a new `kotlin-dsl` plugin from `master` with Kotlin 1.8.10 and the assignment overload support and update the wrapper. I'll clean this up doing this.

Reply:
Ah, got it!

Reply:
Actually I will need to do something in this very PR because Quick Feedback is broken because the `kotlin-dsl` plugin that has assignment overload breaks with Kotlin 1.8.10 :roll_eyes: